content/docs/1.3.0/iofogctl/usage.md

```diff
@@ -250,6 +250,24 @@ or, if we also want to delete the namespace, we can run:
 iofogctl delete namespace zoo-1 --force
 ```
 
+## Check the log output of components
+
+Note: You will need ssh access to the specified resource to use this feature
+
+To check the log output of any resource, use `logs` e.g.
+
+```bash
+Usage:
+  iofogctl logs RESOURCE NAME [flags]
```

Comment:
We haven't pasted iofogctl help into the website anywhere, please don't do here.

The examples below are enough, can remove these 2 lines starting with Usage: